Library/Homebrew/manpages/brew.1.md.erb

```diff
@@ -104,6 +104,7 @@ can take several different forms:
   * An arbitrary file:
     Homebrew can install formulae from a local path. It can point to either a
     formula file or a bottle.
+    Prefix relative paths with `./` to prevent them being interpreted as a tap.
```

Comment:
```suggestion
    Prefix relative paths with `./` to prevent them being interpreted as a formula name or tap.
```

Reply:
Ah, I guess that accounts for when there is no `/` in the path. Will do